<a href="https://colab.research.google.com/github/Abishek0411/Machinge-Learning-Lab/blob/main/ML_exe_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**AIM:**  
We have a dataset named Laptop-Prices from kaggle. Let us devise a program to input, load and view the dataset.

**CODE:**

In [ ]:
# ABISHEK S R
# RA2211003011292

# Importing Libraries
import pandas as pd
import numpy as np

# Uploading dataset
'''
from google.colab import drive
drive.mount('/content/drive')
file_path = '/content/drive/MyDrive/datasets/laptop_prices.csv'
df = pd.read_csv(file_path)
'''
!wget https://raw.githubusercontent.com/Abishek0411/Machinge-Learning-Lab/main/laptop_prices.csv
df = pd.read_csv('laptop_prices.csv')



# Displaying the first few rows of the dataset
print(df.head())




--2024-09-21 18:37:10--  https://raw.githubusercontent.com/Abishek0411/Machinge-Learning-Lab/main/laptop_prices.csv
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 191364 (187K) [text/plain]
Saving to: ‘laptop_prices.csv.1’

laptop_prices.csv.1 100%[===================>] 186.88K  --.-KB/s    in 0.02s   

2024-09-21 18:37:11 (8.56 MB/s) - ‘laptop_prices.csv.1’ saved [191364/191364]

  Company      Product   TypeName  Inches  Ram     OS  Weight  Price_euros  \
0   Apple  MacBook Pro  Ultrabook    13.3    8  macOS    1.37      1339.69   
1   Apple  Macbook Air  Ultrabook    13.3    8  macOS    1.34       898.94   
2      HP       250 G6   Notebook    15.6    8  No OS    1.86       575.00   
3   Apple  MacBook Pro  Ultrabook    15.4   16  macOS    1.83      2537

**Output:**  
--2024-09-21 18:37:10--  https://raw.githubusercontent.com/Abishek0411/Machinge-Learning-Lab/main/laptop_prices.csv
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 191364 (187K) [text/plain]
Saving to: ‘laptop_prices.csv.1’

laptop_prices.csv.1 100%[===================>] 186.88K  --.-KB/s    in 0.02s   

2024-09-21 18:37:11 (8.56 MB/s) - ‘laptop_prices.csv.1’ saved [191364/191364]

  Company      Product   TypeName  Inches  Ram     OS  Weight  Price_euros  \
0   Apple  MacBook Pro  Ultrabook    13.3    8  macOS    1.37      1339.69   
1   Apple  Macbook Air  Ultrabook    13.3    8  macOS    1.34       898.94   
2      HP       250 G6   Notebook    15.6    8  No OS    1.86       575.00   
3   Apple  MacBook Pro  Ultrabook    15.4   16  macOS    1.83      2537.45   
4   Apple  MacBook Pro  Ultrabook    13.3    8  macOS    1.37      1803.60   

     Screen  ScreenW  ...  RetinaDisplay CPU_company CPU_freq      CPU_model  \
0  Standard     2560  ...            Yes       Intel      2.3        Core i5   
1  Standard     1440  ...             No       Intel      1.8        Core i5   
2   Full HD     1920  ...             No       Intel      2.5  Core i5 7200U   
3  Standard     2880  ...            Yes       Intel      2.7        Core i7   
4  Standard     2560  ...            Yes       Intel      3.1        Core i5   

  PrimaryStorage  SecondaryStorage PrimaryStorageType  SecondaryStorageType  \
0            128                 0                SSD                    No   
1            128                 0      Flash Storage                    No   
2            256                 0                SSD                    No   
3            512                 0                SSD                    No   
4            256                 0                SSD                    No   

   GPU_company               GPU_model  
0        Intel  Iris Plus Graphics 640  
1        Intel        HD Graphics 6000  
2        Intel         HD Graphics 620  
3          AMD          Radeon Pro 455  
4        Intel  Iris Plus Graphics 650  

[5 rows x 23 columns]